Source/WPF/MyMoney/Views/TransactionsView.xaml.cs

```diff
@@ -1557,7 +1557,7 @@ private void UpdateView()
                     this.ViewTransfers(this.activeAccount);
                     break;
                 case TransactionViewName.DanglingTransfers:
-                    List<Transaction> dangling = this.myMoney.CheckTransfers();
+                    HashSet<Transaction> dangling = this.myMoney.CheckTransfers();

```

Comment:
this could be `var dangling`

Reply:
why not use strong typing instead of 'var'?
it was strongly typed before. "List<Transaction> dangling"
its there a preference to use Var everywhere?

Reply:
Just results in less code churn - this is a great example where the code would not have to have changed at all if it were originally written using `var`.  But it's a minor point, we don't need to go change very line of code, but it is a pattern I prefer on new code.

Reply:
OK, noted prefer "VAR" over "strong type" declaration

Reply:
fixed